docs/2.4.4-1.2.md

```diff
@@ -0,0 +1,53 @@
+# ProxySQL 2.4.4-1.2 and ProxySQL admin tools (2022-11-0)
+
+<style>
+    table {
+        width=50%';
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | November 2022 |
+|---|---|
+| Install instructions | [ProxySQL and proxysql-admin](https://docs.percona.com/proxysql/install-v2.htm) |
+| Download this version | [ProxySQL 2.x](https://www.percona.com/downloads/proxysql2/) |
+
+[ProxySQL](https://proxysql.com/) is a high-performance proxy for MySQL and MySQL-compatible database servers such as Percona Server for MySQL and MariaDB. It acts as an intermediary for client requests seeking resources from the database. ProxySQL was created for the database administrator as a solution for complex replication topology issues. 
+
+You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL. 
+
+The [ProxySQL Admin (proxysql-admin)](https://docs.percona.com/proxysql/v2-config.html) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](https://docs.percona.com/proxysql/proxysql-v1.html) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](https://docs.percona.com/proxysql/psql-functions.html) supports Percona XtraDB Cluster without custom scripts.
+
+The [Percona Scheduler Admin (percona-scheduler-admin)](https://docs.percona.com/proxysql/psa-scheduler.html) tool can automatically perform a failover due to node failures, service degradation, or maintenance.
+
+!!! note
```

Comment:
This note is in several places in the docs. 
Let's remove the note heading and add the text to the PSA paragraph.

Reply:
Updated